### PR TITLE
Update http3.md

### PR DESCRIPTION
**Pull request by samsp-msft**
Fixed a typo and added a bit more context.

***@Rick-Anderson*** edit: [Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel/http3?view=aspnetcore-6.0&branch=pr-en-us-23001)

### CODE REVIEW

File: aspnetcore/fundamentals/servers/kestrel/http3.md [hunk 1]
```diff
@@ -21,15 +21,17 @@ HTTP/3 and QUIC have a number of benefits compared to HTTP/1.1 and HTTP/2:
...
+> HTTP/3 is available in .NET 6 as a preview feature as the HTTP/3 specification hasn't been finalized and there may be behavioral or performance issues in HTTP/3 with .NET 6.
 >
-> Apps that want take advantage of HTTP/3 should be designed to support all 3 versions versions. If issues are identified in HTTP/3 then it is recommended that apps temporarily disable it until issues are resolved and the app can be updated.
+> Apps that want take advantage of HTTP/3 should be designed to support all 3 HTTP versions. If issues are identified in HTTP/3 then it is recommended that apps temporarily disable it until issues are resolved and the app can be updated.
 
 ## HTTP/3 requirements
 
-HTTP/3 isn't enabled by default. Because not all routers, firewalls, and proxies properly support HTTP/3, we recommend configuring HTTP/3 together with HTTP/1.1 and HTTP/2. This can be done by specifying `HttpProtocols.Http1AndHttp2AndHttp3` as an endpoint's supported protocols.
+As HTTP/3 support is in preview, it is not enabled by default.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/servers/kestrel/http3.md:30]:
```suggestion
HTTP/3 support is in preview, therefore it's not enabled by default.
```

File: aspnetcore/fundamentals/servers/kestrel/http3.md [hunk 2]
```diff
@@ -21,15 +21,17 @@ HTTP/3 and QUIC have a number of benefits compared to HTTP/1.1 and HTTP/2:
...
 
-If the platform that Kestrel is running on doesn't have all the requirements for HTTP/3 then it is automatically disabled.
+Because not all routers, firewalls, and proxies properly support HTTP/3, we recommend configuring HTTP/3 together with HTTP/1.1 and HTTP/2. This can be done by specifying `HttpProtocols.Http1AndHttp2AndHttp3` as an endpoint's supported protocols.
+
+HTTP/3 uses QUIC as its transport protocol. The .NET implementation of HTTP/3 uses [MsQuic](https://github.com/microsoft/msquic) to provide QUIC functionality. MSQuic is included in specific builds of windows and as a library for linux. If the platform that Kestrel is running on doesn't have all the requirements for HTTP/3 then it is automatically disabled.
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/servers/kestrel/http3.md:34]:
```suggestion
HTTP/3 uses QUIC as its transport protocol. The .NET implementation of HTTP/3 uses [MsQuic](https://github.com/microsoft/msquic) to provide QUIC functionality. MSQuic is included in specific builds of windows and as a library for Linux. If the platform that Kestrel is running on doesn't have all the requirements for HTTP/3 then it's disabled.
```

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by Rick-Anderson** [comment]
@samsp-msft  can you accept/reject my suggestions or give me permission to apply them?

**Comment by samsp-msft** [comment]
Applied your suggestions. The checkbox for allowing edits by maintainers is checked, do I need to do something else to give you permissions?

**Comment by Rick-Anderson** [comment]
> Applied your suggestions. The checkbox for allowing edits by maintainers is checked, do I need to do something else to give you permissions?

Nope, I just didn't want to apply them without your permission. Thanks

**Comment by Rick-Anderson** [comment]
@samsp-msft  thanks for the PR, really nice improvements

**Pull request merged**